matplotlib_scalebar/test_scalebar.py

```diff
@@ -33,7 +35,7 @@ def scalebar():
 
     yield scalebar
 
-    plt.draw()
```

Comment:
I think the draw is important here to make sure the scale bar is drawn. It doesn't look that it will impact your unit test